index.js

```diff
@@ -1,17 +1,28 @@
 "use strict";
 
 function assignSender(context) {
-  context.github.issues.addAssigneesToIssue(context.issue({
-    assignees: [context.payload.sender.login]
-  }));
+  context.github.issues.addAssigneesToIssue(
+    context.issue({
+      assignees: [context.payload.sender.login],
+    })
+  );
 }
 
 module.exports = (robot) => {
+  robot.on(
+    ["issues.opened", "issues.edited", "issues.unassigned"],
+    async (context) => {
+      const config = await context.config("config.yml", {
+        addAssignees: true,

```

Comment:
If we rename the file, we should name this to `enabled`
```suggestion
        enabled: true,

```